### PR TITLE
chore: Fix find_by_images_tests.py

### PR DESCRIPTION
**Pull request by ki4070ma**
For https://github.com/appium/python-client/issues/373

### CODE REVIEW

File: test/functional/android/search_context/find_by_image_tests.py [hunk 1]
```diff
@@ -13,23 +13,25 @@
 # limitations under the License.
 
 import base64
+import os
```
Comment (ki4070ma) [inline test/functional/android/search_context/find_by_image_tests.py]: remove unused imports.

File: ci-jobs/functional/run_appium.yml [hunk 2]
```diff
@@ -5,6 +5,9 @@ steps:
   displayName: Install Node 11.x
 - script: npm install -g appium@beta --chromedriver_version='2.44'
   displayName: Install appium
+- script: npm install -g opencv4nodejs
```
Comment (mykola-mokhnach) [inline ci-jobs/functional/run_appium.yml]: how much longer the tests take now?

Comment (ki4070ma) [reply ci-jobs/functional/run_appium.yml]: This installation step takes 8m46s.
In total, it's 20min.
https://dev.azure.com/ki4070ma/python-client/_build/results?buildId=1046&view=logs

Comment (mykola-mokhnach) [reply ci-jobs/functional/run_appium.yml]: what was the previous value (before enabling opencv)?

Comment (ki4070ma) [reply ci-jobs/functional/run_appium.yml]: It's 10m30s previously.
Almost just added installation time.
https://dev.azure.com/ki4070ma/python-client/_build/results?buildId=1047

Comment (mykola-mokhnach) [reply ci-jobs/functional/run_appium.yml]: I'm ok with the change if such increase in tests execution duration works for you

**Pull request merged**